packages/flutter/test/widgets/single_child_scroll_view_test.dart

```diff
@@ -107,6 +107,49 @@ void main() {
     expect(context.clipBehavior, equals(Clip.hardEdge));
   });
 
+  testWidgets('SingleChildScrollView ScrollViewKeyboardDismissBehavior.onDrag test',
+      (WidgetTester tester) async {
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          body: SingleChildScrollView(
+            keyboardDismissBehavior: ScrollViewKeyboardDismissBehavior.onDrag,
+            child: Column(
+              children: <Widget>[
+                Autocomplete<String>(
+                  optionsBuilder: (TextEditingValue textEditingValue) {
+                    return <String>['One', 'Two', 'Three'].where((String option) {
+                      return option.contains(textEditingValue.text.toLowerCase());
+                    }).toList();
+                  },
+                  fieldViewBuilder: (BuildContext context, TextEditingController textEditingController, FocusNode focusNode, VoidCallback onFieldSubmitted) {
+                    return TextField(
+                      controller: textEditingController,
+                      focusNode: focusNode,
+                    );
+                  },
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+    final Finder textFieldFinder = find.byType(TextField);
+    expect(textFieldFinder, findsOneWidget);
+
+    await tester.tap(textFieldFinder);
+    await tester.pump();
+
+    final TextField textField = tester.widget(textFieldFinder);
+    expect(textField.focusNode!.hasFocus, isTrue);
+
+    await tester.drag(textFieldFinder, const Offset(0, 500));
+    await tester.pumpAndSettle();
+
+    expect(textField.focusNode!.nextFocus(), isTrue);
```

Comment:
Why not just expect hasFocus isFalse?

Reply:
focusNode is assigned to TextField, and i think it won't loose its focus until unless `onTapoutside` is used, therefore we can look for the focus is going to next widget.

Reply:
As-is, this test passes on master and doesn't seem to test the change in this PR.